Linting issues

```
op-chain-ops/cmd/op-upgrade/main.go:73: File is not `goimports`-ed (goimports)
				Name:     "deploy-config",
				Usage:    "The path to the deploy config file",
				EnvVars:  []string{"DEPLOY_CONFIG"},
op-chain-ops/upgrades/l1.go:622:3: ineffectual assignment to startBlock (ineffassign)
		startBlock := common.Hash{}
		^
make: *** [Makefile:16: lint-go] Error 1

Exited with code exit status 2
```

--- a/op-chain-ops/cmd/op-upgrade/main.go
+++ b/op-chain-ops/cmd/op-upgrade/main.go
@@ -148,7 +148,7 @@ func entrypoint(ctx *cli.Context) error {
 		name, _ := toDeployConfigName(chainConfig)
 		config, err := genesis.NewDeployConfigWithNetwork(name, deployConfig)
 		if err != nil {
-			log.Warn("Cannot find deploy config for network", "name", chainConfig.Name, "deploy-config-name", name, "path", deployConfig, "err", err)
+			log.Warn("Cannot find deploy config for network, so validity checks will be skipped", "name", chainConfig.Name, "deploy-config-name", name, "path", deployConfig, "err", err)
 		}
 
 		if config != nil {

--- a/op-chain-ops/cmd/op-upgrade/main.go
+++ b/op-chain-ops/cmd/op-upgrade/main.go
@@ -70,9 +70,9 @@ func main() {
 				EnvVars: []string{"SUPERCHAIN_TARGET"},
 			},
 			&cli.PathFlag{
-				Name:     "deploy-config",
-				Usage:    "The path to the deploy config file",
-				EnvVars:  []string{"DEPLOY_CONFIG"},
+				Name:    "deploy-config",
+				Usage:   "The path to the deploy config file",
+				EnvVars: []string{"DEPLOY_CONFIG"},
 			},
 			&cli.PathFlag{
 				Name:    "outfile",

--- a/op-chain-ops/upgrades/l1.go
+++ b/op-chain-ops/upgrades/l1.go
@@ -619,8 +619,7 @@ func SystemConfig(batch *safe.Batch, implementations superchain.ImplementationLi
 			return err
 		}
 
-		startBlock := common.Hash{}
-
+		var startBlock common.Hash
 		if config != nil {
 			startBlock = common.BigToHash(new(big.Int).SetUint64(config.SystemConfigStartBlock))
 		} else {

